src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAX_NUMBER = 100;
+
+    public Ball getRandomBall() {
+        return new Ball(new ColorSupplier().getRandomColor(),
+                new Random().nextInt(MAX_NUMBER + 1));
```

Comment:
Make Random and ColorSupplier consant variables of class.
Also, better to use SecureRandom implementation.